main.go

```diff
@@ -50,9 +50,11 @@ func init() {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
+	var namespace string
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&namespace, "namespace", "", "The namespace that the manager's cache can watch objects in.")
```

Comment:
nit: This wording sounds a bit ambiguous. Maybe `The namespace in which the manager controls and reconciles resources`?

Reply:
I actually picked it up from some k8s documentation i think

Reply:
It's not even grammatically correct haha. That's awkward

Reply:
Fixed.